config/traefik/traefik.enterprise.yaml

```
@@ -4,37 +4,104 @@ http:
   # Routers
   #
   routers:
-
     #
     # tenantadm
     #
     tenantadm:
       entrypoints: https
       middlewares: #{{- block "circuit-breaker" . -}}{{- if not (env "TESTING") }}
-      - circuit-breaker #{{- end}}{{end}}
-      - userauth
-      - sec-headers
-      - compression
+        - circuit-breaker #{{- end}}{{end}}
+        - userauth
+        - sec-headers
+        - compression
       rule: "PathPrefix(`/api/management/{version:v[0-9]+}/tenantadm`)"
       service: tenantadm
       tls: true
 
     tenantadmSignup:
       entrypoints: https
       middlewares: #{{- template "circuit-breaker"}}
-      - sec-headers
-      - compression
+        - sec-headers
+        - compression
       rule: >-
         Method(`OPTIONS`,`POST`) && Path(`/api/management/{version:v[0-9]+}/tenantadm/tenants`) ||
         Method(`OPTIONS`,`POST`) && Path(`/api/management/{version:v[0-9]+}/tenantadm/tenants/trial`)
       service: tenantadm
       tls: true
 
+    #
+    # auditlogs
+    #
+    auditlogs:
+      entrypoints: https
+      middlewares: #{{- if not (env "TESTING") }}
+        - circuit-breaker #{{- end}}
```

(Nitpick) can be expressed more compactly using the above template block:
```suggestion
      middlewares: #{{- template "circuit-breaker"}}
```

---

Did that, thanks!